chromadb/cli/cli.py

```diff
@@ -31,6 +31,9 @@ def run(
     path: str = typer.Option(
         "./chroma_data", help="The path to the file or directory."
     ),
+    persistent: Annotated[
```

Comment:
I think this would be confusing at best. Wouldn't it be more appropriate not to set `--path`?

Imagine `chroma run --path <path> --persistent False`.

In the above, `--path` is still mandatory, which wouldn't help much either.

Reply:
I done it so that it wont brake compatibility. There is a chance that some people running it without providing the path parameter so that it defaults to "./chroma_data".

Reply:
The path param is optional you can run it by just `chroma run`

Reply:
Hmm, or you mean throwing an error if the user sets the path parameter?

Reply:
Fair point but I still feel that 

```bash
chroma run --path ./chroma --no-persistent
```

can be confusing.

I think that from CLI perspective having less flags is better than more.  And introducing a breaking change to have no path translate to ephemeral instance. The absence of something can be a better mechanic than default to something the user might not expect.

The CLI is still nascent so we might not want to be overzealous on backward compatibility of defaults. I don't think we have telemetry on how frequently or not people are using the CLI to run Chroma so it might be a shot in the dark.

@jeffchuber any thoughts on this?